ext/winevt/winevt_query.c

```diff
@@ -111,6 +131,23 @@ rb_winevt_query_initialize(VALUE argc, VALUE *argv, VALUE self)
     }
   }
 
+  switch (TYPE(rb_flags)) {
+  case T_SYMBOL:
+    flags = get_evt_query_flag_from_cstr(RSTRING_PTR(rb_sym2str(rb_flags)));
+    break;
+  case T_STRING:
+    flags = get_evt_query_flag_from_cstr(StringValuePtr(rb_flags));
```

Comment:
Are string & symbol needed?
It seems that they doesn't support combination of multiple flags, and there names are slightly different from original enum.
It would be better to delete them than to implement them with incomplete specifications, only supporting `Fixnum` seems enough for now.

Reply:
Once I started to implement this feature, I wanted to write like as:

```ruby
@query = Winevt::EventLog::Query.new("/path/to/channel_stored.evtx", "*", @session, :file)
```

Also, `EvtQueryTolerateQueryErrors` is making a room to process loosen XPath conditions.
So, it would be reasonable to handle `:file` or `:channel` to represent flags of ExtQuery.

Reply:
Hmm..., I understand what you mean. Yes, only supporting enum should be fine.